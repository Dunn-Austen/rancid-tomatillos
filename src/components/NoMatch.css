.no-match {
  align-items: center;
  background: linear-gradient(rgba(0, 0, .5, .5), rgba(0, .1, .1, .1)), url("../images/curtains.jpeg");
  background-blend-mode: lighten saturation;
  background-repeat: no-repeat;
  background-size: 100% 100%;
  border: 5px double #C654DE;
  box-sizing: border-box;
  color: #D45DC1;
  display: flex;
  flex-direction: column;
  height: 92vh;
  justify-content: space-around;
  margin: 0px;
  overflow: scroll;
}

.invalid-route-header {

}

.invalid-route-button {

}
