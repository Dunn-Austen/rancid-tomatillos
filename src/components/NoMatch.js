import React from 'react';
import { Link } from 'react-router-dom';


const NoMatch = () => {
  return (
    <div className='no-match'>
      <h2 className='invalid-route-header'>Uh-oh! Page Not Found</h2>
      <Link to ='/'>
        <button className='invalid-route-button'>Return To Homepage</button>
      </Link>
    </div>
  )
}

export default NoMatch;
