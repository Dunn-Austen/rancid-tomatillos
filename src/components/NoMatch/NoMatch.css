.no-match {
  align-items: center;
  background: linear-gradient(rgba(0, 0, .5, .5), rgba(0, .1, .1, .1)), url("../../images/curtains.jpeg");
  background-blend-mode: lighten saturation;
  background-repeat: no-repeat;
  background-size: 100% 100%;
  border: 5px double #C654DE;
  box-sizing: border-box;
  color: #D45DC1;
  display: flex;
  flex-direction: column;
  height: 92vh;
  margin: 0px;
  overflow: scroll;
}

.invalid-route-header {
  font-size: 3em;
  margin: 120px 120px 55px 120px;
  text-align: center;
}

.invalid-route-button {
  background-color: #34032f;
  border: 0.1px groove #c554dd;
  border-radius: 3px;
  box-shadow: 0 0 1px 1px white;
  color: white;
  font-size: 20px;
  font-weight: bold;
  height: 45px;
  line-height: 5px;
  outline: none;
  width: 221px;
}

.invalid-route-button:hover {
  background-color: #c654dd;
  border: 1px groove black;
  color: white;
}
