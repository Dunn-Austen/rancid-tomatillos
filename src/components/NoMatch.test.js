import React from 'react';
import { shallow } from 'enzyme';
import NoMatch from './NoMatch';

describe('NoMatch', () => {
  let wrapper;

  it('should match snapshot', () => {
    wrapper = shallow(<NoMatch />);
    expect(wrapper).toMatchSnapshot();
  })
})
