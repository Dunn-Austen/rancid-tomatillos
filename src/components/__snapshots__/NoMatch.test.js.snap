// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch should match snapshot 1`] = `
<div
  className="no-match"
>
  <h2
    className="invalid-route-header"
  >
    Uh-oh! Page Not Found
  </h2>
  <Link
    to="/"
  >
    <button
      className="invalid-route-button"
    >
      Return To Homepage
    </button>
  </Link>
</div>
`;
